orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -1560,6 +1565,7 @@ def run_step(model1, model2, model3, x1, x2):
         _test_helpers.assert_gradients_match_and_reset_gradient(ort_model1, pt_model1, atol=2e-6)
         _test_helpers.assert_gradients_match_and_reset_gradient(ort_model2, pt_model2)
         _test_helpers.assert_gradients_match_and_reset_gradient(ort_model3, pt_model3)
+    assert training_ortmodule.ONNX_OPSET_VERSION == DEFAULT_OPSET
```

Comment:
Instead of adding this line to many (unrelated) tests, may we just add a single unit test to check the default onnx opset version?

Reply:
I'll change. I did it that way to avoid creating a new test and be faster.